packages/web/components/PageHeader.tsx

```diff
@@ -41,63 +93,119 @@ export const PageHeader: React.FC = () => {
       justify="space-between"
       wrap="wrap"
       color="offwhite"
-      py="6"
+      py="4"
       px="8"
+      position="relative"
+      display={{ base: 'none', md: 'flex' }}
     >
       <MetaLink href="/" display="block" mr="10">
         <Image src={MetaGameImage} alt="MetaGame" mt={-2} w="9rem" />
       </MetaLink>
 
-      <Button
-        variant="link"
-        display={{ base: 'block', md: 'none' }}
-        onClick={handleToggle}
+      <Stack
+        width="48rem"
+        height="100%"
+        direction="row"
+        justify="center"
+        align="center"
+        position="absolute"
+        top="0"
+        left="calc(50% - 24rem)"
+        padding="0 0 0.5rem 0"
       >
-        <svg
-          fill="white"
-          width="1.5rem"
-          viewBox="0 0 20 20"
-          xmlns="http://www.w3.org/2000/svg"
+        <Image
+          src={MetaDrawer}
+          alt="MetaDrawer"
+          w="48rem"
+          position="absolute"
+          left="calc(50% - 24rem)"
+          top="0"
+          zIndex="1"
+        />
+
+        <MenuItem
+          href="https://forum.metagame.wtf/c/quest/5/l/latest?board=default"
```

Comment:
This quest board isnt really used much. I think for the main items in this nav, it should have a link to the Wiki, a link to buy SEED, a link to the forums and a link to the newsletter